front/src/app/core/services/globalclock/globalclock.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { FixtureService } from '../fixture/fixture.service';
+import { ArtnetService } from '../artnet/artnet.service';
+
+@Injectable()
+export class GlobalclockserviceService {
+  private timer: NodeJS.Timer;
+  private fixtureService: FixtureService;
+  private artnetService: ArtnetService;
+
+
+  constructor(fixtureService: FixtureService, artnetService: ArtnetService) {
```

Comment:
you can declare the arguments as private, then you don't need to define them as extra variables!